FantasmoSDK/Classes/Utilities/ObservationToken.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ObservationToken.swift
+//  FantasmoSDK
+//
+//  Created by Yauheni Klishevich on 19.07.2021.
+//
+
+import Foundation
+
+public class ObservationToken {
```

Comment:
This class is a bit cryptic. Can you add some comments here?

Reply:
I have added only yesterday, just didn't report this here